pkgs/development/tools/pnpm/fetch-deps/default.nix

```
@@ -32,79 +33,84 @@
             outputHash = "";
             outputHashAlgo = "sha256";
           };
-      installFlags = lib.optionalString (pnpmWorkspace != "") "--filter=${pnpmWorkspace}";
+
+      filterFlags = lib.map (package: "--filter=${package}") pnpmWorkspaces;
     in
-    stdenvNoCC.mkDerivation (
-      finalAttrs:
+    lib.throwIf (args.pnpmWorkspace != null)
```

This was the other eval error: `error: attribute 'pnpmWorkspace' missing`
```suggestion
    lib.throwIf (builtins.hasAttr "pnpmWorkspace" args)
```

---

`args ? pnpmWorkspace`

---

Fixed